libs/shared/src/main/java/io/crate/common/Ints.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.common;
+
+public class Ints {
+
+    private Ints() {
+    }
+
+    public static Integer tryParse(String string) {
```

Comment:
Maybe worth checking in a follow up if we can remove this. It's only used in `fetchSummits` and the data their should parse without errors.

Reply:
Tests are failing on empty string values in tests, so i moved `tryParse` into `SummitsIterable` for now.